os-eap-migration/added/launch/openshift-migrate-common.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+# Openshift EAP common migration script
+
+source ${JBOSS_HOME}/bin/launch/openshift-common.sh
+source ${JBOSS_HOME}/bin/probe_common.sh
+source /opt/partition/partitionPV.sh
+
+function runMigration() {
+  local instanceDir=$1
+  local count=$2
+
+  export NODE_NAME="${NODE_NAME:-node}-${count}"
+  cp -f ${STANDALONE_XML_COPY} ${STANDALONE_XML}
+
+  source $JBOSS_HOME/bin/launch/configure.sh
+
+  echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
+
+  local txOptions="-Dcom.arjuna.ats.arjuna.common.RecoveryEnvironmentBean.recoveryBackoffPeriod=1 -Dcom.arjuna.ats.arjuna.common.RecoveryEnvironmentBean.periodicRecoveryPeriod=1 -Dcom.arjuna.ats.jta.JTAEnvironmentBean.orphanSafetyInterval=1"
+  local terminatingFile="${JBOSS_HOME}/terminatingMigration"
+
+  (runMigrationServer "$instanceDir" "${txOptions}") &
+
+  PID=$!
+
+  rm -f "${terminatingFile}"
+
+  trap "echo Received TERM ; touch \"${terminatingFile}\" ; kill -TERM $PID ; " TERM
+  local success=false
+  local message="Finished, migration pod has been terminated"
+  ${JBOSS_HOME}/bin/readinessProbe.sh
+
+  if [ $? -eq 0 ] ; then
+    echo "$(date): Server started, checking for transactions"
+    local startTime=$(date +'%s')
+    local endTime=$((startTime + ${RECOVERY_TIMEOUT} + 1))
+
+    local socketBinding=$(run_cli_cmd '/subsystem=transactions/:read-attribute(name="socket-binding")' | grep -w result | sed -e 's+^.*=> "++' -e 's+".*$++')
+    local recoveryPort=$(run_cli_cmd '/socket-binding-group=standard-sockets/socket-binding='"${socketBinding}"'/:read-attribute(name="bound-port")' | grep -w result | sed -e 's+^.*=> ++')
+    local recoveryHost=$(run_cli_cmd '/socket-binding-group=standard-sockets/socket-binding='"${socketBinding}"'/:read-attribute(name="bound-address")' | grep -w result | sed -e 's+^.*=> "++' -e 's+".*$++')
+
+    if [ -n "${recoveryPort}" ] ; then
```

Comment:
if the recovery-listener is not enabled, it would be literally "undefined", so not empty

Reply:
@maschmid Thanks, will update this now